bin/test-workflow/QUICKSTART.md

```diff
@@ -0,0 +1,161 @@
+# Conjur Kubernetes Authentication Quick Start Guide
+
+The following guide details an end-to-end workflow for using Conjur Kubernetes
+authentication by deploying a PetStore demo application to a
+Kubernetes-in-Docker (kind) cluster.
+
+At the end of this quickstart, you will have:
+
+* Deployed Conjur OSS to a kind cluster
```

Comment:
This is purely an observation comment and you don't need to act on it other than potentially creating an issue for future work. 

It's definitely simpler at this point to be prescriptive in this guide and mention only using the KinD cluster. However, in future work it would be nice if the KinD step became optional so that users could bring their own Kubernetes cluster. The requirement being that `kubectl` is already setup to communicate to their cluster of choice. This would change the steps of the quickstart from:


* Deployed Conjur OSS to a kind cluster
* ...

to:

* Created a KinD cluster (optional)
* Deployed Conjur OSS to a Kubernetes cluster
* ...

Reply:
Filed #329